README.md

```diff
@@ -1,52 +1,64 @@
+**Codius Manifest** (this repository) is a module for validating and generating Codius manifests.
 The Codius manifest format allows users to specify container images, public and
 private environment variables, and other information about pods. Manifests are used
 by [Codius hosts](https://github.com/codius/codiusd) to setup the container environments and download images.
 
-## Manifest Example
-Manifests must match the standard format, which is specified [here](https://github.com/codius/manifest/blob/master/src/schemas/ManifestSpec.json).
+## Manifest Format
+Manifests must match the standard format, which is specified [here](https://github.com/codius/manifest/blob/master/src/schemas/GeneratedManifestSpec.json).
+Manifests that are valid against the standard schema are considered complete.
 ```json
 {
-  "name": "test-app",
-  "version": "1.0.0",
-  "machine": "small",
-  "port": "8080",
-  "containers": [{
-    "id": "app",
-    "image": "hello-world@sha256:f5233545e43561214ca4891fd1157e1c3c563316ed8e237750d59bde73361e77",
-    "command": ["/bin/sh"],
-    "workdir": "/root",
-    "environment": {
-      "AWS_ACCESS_KEY": "$AWS_ACCESS_KEY",
-      "AWS_SECRET_KEY": "$AWS_SECRET_KEY"
+  "manifest": {
+    "name": "my-codius-pod",
+    "version": "1.0.0",
+    "machine": "small",
+    "containers": [{
+      "id": "app",
+      "image": "hello-world@sha256:f5233545e43561214ca4891fd1157e1c3c563316ed8e237750d59bde73361e77",
+      "command": ["/bin/sh"],
+      "workdir": "/root",
+      "environment": {
+        "AWS_ACCESS_KEY": "$AWS_ACCESS_KEY",
+        "AWS_SECRET_KEY": "$AWS_SECRET_KEY"
+      }
+    }],
+    "vars": {
+      "AWS_ACCESS_KEY": {
+        "value": "AKRTP2SB9AF5TQQ1N1BB"
+      },
+      "AWS_SECRET_KEY": {
+        "encoding": "private:sha256",
+        "value": "95b3449d5b13a4e60e5c0218021354c447907d1762bb410ba8d776bfaa1a3faf"
+      }
     }
-  }],
-  "vars": {
-    "AWS_ACCESS_KEY": {
-      "value": "AKRTP2SB9AF5TQQ1N1BB"
-    },
-    "AWS_SECRET_KEY": {
-      "encoding": "private:sha256",
-      "value": "d429d080fa3a4629afb7d0759640b60f8ac70b377c72c048cdff9e5ef73dd6c2"
+  },
+  "private": {
+    "vars": {
+      "AWS_SECRET_KEY": {
+        "nonce": "123450325",
+        "value": "AKRTP2SB9AF5TQQ1N1BC"
+      }
     }
   }
 }
 ```
 
-## Full Codius Request Body
-The full Codius request body may include private variables that should not be exposed publicly by the host.
-
+## Codius Files
+Manifests are generated from two files: `codius.json` and `codiusvars.json`.
+### `codius.json`
+This file includes details about the pod to be uploaded . Unlike the generated manifest, `codius.json`
+may contain description fields for public variables. The official specification can be found
+[here](https://github.com/codius/manifest/blob/master/src/schemas/CodiusSpec.json).
 ```json
 {
   "manifest": {
-    "name": "test-app",
+    "name": "my-codius-pod",
     "version": "1.0.0",
     "machine": "small",
-    "port": "8080",
```

Comment:
whats the reason behind removing the port here?

Reply:
The port should actually still be there. Gonna update this.